index.html

```diff
@@ -1492,6 +1492,62 @@ <h3>Types</h3>
 
       </section>
 
+      <section>
+        <h3>Names and Descriptions</h3>
+
+        <p>
+When displaying a <a>credential</a>, it can be useful to have
+text provided by the <a>issuer</a> that provides the
+<a>credential</a> with a name as well as a short description of the
+purpose of the <a>credential</a>. The `name` and `description` <a>properties</a>
+are provided for these purposes.
+        </p>
+
+        <dl>
+          <dt><dfn class="lint-ignore">name</dfn></dt>
+          <dd>
+An OPTIONAL property that expresses the name of the <a>credential</a>. If
+present, the value of the <code>name</code> <a>property</a> MUST be a string.
+Ideally, the name of a <a>credential</a> is concise, human-readable, and could
+be used to enable an individual to quickly differentiate the
+<a>credential</a> from other <a>credentials</a> that they might hold.
+          </dd>
+          <dt><dfn class="lint-ignore">description</dfn></dt>
+          <dd>
+An OPTIONAL property that conveys specific details about a <a>credential</a>.
+If present, the value of the <code>description</code> <a>property</a> MUST be a
+string. Ideally, the description of a <a>credential</a> is no longer than a
+few sentences in length and conveys enough information about the
+<a>credential</a> to remind an individual as to its contents without having
+to look through the entirety of the <a>claims</a>.
+          </dd>
+        </dl>
+
+        <pre class="example nohighlight vc"
+          title="Usage of the name and description property"
+          data-vc-vm="https://university.example/issuers/565049#key-1">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "http://university.example/credentials/3732",
+  <span class="highlight">"type": ["VerifiableCredential", "ExampleDegreeCredential"]</span>,
+  "issuer": "https://university.example/issuers/565049",
```

Comment:
```suggestion
  "issuer": {
    "id": "https://university.example/issuers/565049",
    "name": "Example University",
    "description": "🧑‍🎓 Dàxué"
  },
```

Reply:
What does "🧑‍🎓 Dàxué" mean? I'm fine w/ expanding the issuer to include name and description, but the description should be something that's not going to confuse readers.

Reply:
Fixed in 390360d0f3a2cf5f8f3d4510e8e96f5d8650ca51.